tests/e2e/test_app.py

```diff
@@ -19,51 +19,51 @@ def driver():
     with webdriver.Chrome('./chromedriver', options=opts) as driver:
         yield driver
 
-@pytest.fixture(scope='module')
```

Comment:
it looks like Trello have changed their API, breaking this test in-between me doing this module and the markups. Given that module 9 removes trello, I'm not going to bother fixing it

Reply:
As per comment on M7 - this test still passes locally for me (with some tweaks to list names) so I would question if this may actually be an expired API key? But happy for this to be disabled given M9 has been raised and reactivates the test